daemon/daemon.go

```diff
@@ -109,6 +111,8 @@ type Daemon struct {
 	defaultLogConfig runconfig.LogConfig
 	RegistryService  *registry.Service
 	EventsService    *events.Events
+	uidMaps          []idtools.IDMap
```

Comment:
Yeah, actually I don't see why we can't use `libcontainer/configs` in pkg. It looks horrible. We need to find those folks for serious talk.